common/djangoapps/enrollment/handlers.py

```diff
@@ -0,0 +1,67 @@
+"""
+Handlers and actions related to enrollment.
+"""
+import logging
+from smtplib import SMTPException
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.core.mail.message import EmailMessage
+from django.dispatch import receiver
+from edxmako.shortcuts import render_to_string
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+
+from student.models import ENROLL_STATUS_CHANGE, EnrollStatusChange
+
+LOGGER = logging.getLogger(__name__)
+
+
+@receiver(ENROLL_STATUS_CHANGE)
+def send_email_to_staff_on_student_enrollment(sender, event=None, user=None, **kwargs):  # pylint: disable=unused-argument
+    """
+    Sends an e-mail to staff after a new enrollment.
+    This feature can be enabled by setting the e-mail of the staff in ENROLLMENT_NOTIFICATION_EMAIL in lms.env.json,
+    or by using a SiteConfiguration variable of the same name (which will override the env one).
+    Disabled by default.
+    """
+
+    if event == EnrollStatusChange.enroll:
+        to_email = configuration_helpers.get_value('ENROLLMENT_NOTIFICATION_EMAIL',
+                                                   settings.ENROLLMENT_NOTIFICATION_EMAIL)
+
+        if not to_email:
+            # feature disabled
+            return
+
+        course_id = kwargs['course_id']
+
+        site_protocol = 'https' if settings.HTTPS == 'on' else 'http'
+        site_domain = configuration_helpers.get_value('site_domain', settings.SITE_NAME)
+        context = {
+            'user': user,
+            # This full_name is dependent on edx-platform's profile implementation
+            'user_full_name': user.profile.name if hasattr(user, 'profile') else None,
+
+            'course_url': u"{protocol}://{domain}{path}".format(
```

Comment:
Consider using [urlunsplit](https://docs.python.org/2.7/library/urlparse.html?highlight=urlunsplit#urlparse.urlunsplit), so you don't have to worry about a slash existing between `domain` and `path`.

Reply:
Ok, done. I also tested that if the path contains a `/`, it doesn't add one.